src/modules/site/base/views/user.py

```diff
@@ -68,3 +105,22 @@ def user_update():
     user.save()
     return redirect(url_for('user.user_profile'))
   return render_template('user/update.html', **locals())
+
+
+@user_bp.route("/password/reset", methods=["GET", "POST"])
+@jwt_required()
+def user_reset_password():
+  title = "Reset Password"
+  disable_parent_breadcrumb = True
+  user = get_current_user()
+  form = PasswordResetForm(request.form)
+
+  if request.method == 'POST' and form.validate():
+    password = slugify(request.form.get("password"))

```

Comment:
does the form.validation assert that password and password-verify match on the server-side?

Reply:
Yes, it applies the any validators set when creating the FlaskForm on the form data that was posted.